source/mongodump.txt

```diff
@@ -25,18 +25,22 @@
 Synopsis
 --------
 
-:binary:`~bin.mongodump` is a utility for creating a binary export of
-the contents of a database. :binary:`~bin.mongodump` can export data
-from either :binary:`~bin.mongod` or :binary:`~bin.mongos` instances;
-i.e. can export data from standalone, replica set, and sharded cluster
-deployments.
+:binary:`~bin.mongodump` is a utility that creates a binary export of
+a database's contents.
+
+:binary:`~bin.mongodump` can export data from:
+
+- :binary:`~bin.mongod` and :binary:`~bin.mongos` instances
```

Comment:
Having "mongod and mongos" in the same list as the cluster topologies seems weird to me. It's saying the same thing in a different way, no? Maybe we can move this to a sentence after the list of topologies to let people know that they can connect to both mongods and mongoses?

Reply:
Done